src/components/svg-wrapper.tsx

```diff
@@ -267,6 +328,55 @@ const SvgWrapper = () => {
             onKeyDown={handleKeyDown}
         >
             <SvgCanvas />
+            {mode === 'select' && selectStart.x != 0 && selectStart.y != 0 && (
+                <g id="select_in_progress___no_use" transform={`translate(${calcSelectSX()}, ${calcSelectSY()})`}>
+                    <rect
```

Comment:
IMO, We may merge those into one `<rect stroke="xxx" strokeWidth="2" fill="xxx" opacity="xxx" />`.